beartype/_cave/abc.py

```diff
@@ -41,6 +41,7 @@
 
 # ....................{ IMPORTS                           }....................
 from abc import ABCMeta, abstractmethod
+from typing import Union
```

Comment:
@leycec, I'll take this opportunity to raise a point about the future state of the `Union` type  - As per [PEP 604](https://www.python.org/dev/peps/pep-0604/), in Python 3.10 half of a C or statement - `|` - essentially aliases the Union type. I.E.:

```
def bear_func(param1: str, param2: int) -> Union[str, int]:....
```
will be equivalent to:
```
def bear_func(param1: str, param2: int) -> str | int: ....
```
Most of the work I've done has been on the data structure and testing side so I'm less familiar with the actual type hinting implementation. In short, I need to research the aforementioned implementation before opening a ticket on the matter. Barring unforeseen events, the plan was to tackle this over the weekend along with #24.

Reply:
```
def bear_func(param1: str, param2: int) -> str | int: ....
```

Freyja above, but that's *really* smokin' hot. If there's one thing we need, it's money and ~~drugs~~ vaccines. If there's two things we need, it's more set, group, and category theory operators pervading every aspect of our all-too-brief lives as shambolic meat suits on this pale blue dot.

Thanks as always for the apropos heads up, Master Tyler. I had no idea PEP 604 was even a thing, but now I do and I'm a better Canadian man for it.